WebView/app/src/main/assets/main.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 function sendAndroidMessage() {
-	Weather.sendMessage("The weather in " + `${document.getElementById("title").innerText}` + " today is " + 
+	jsObject.postMessage("The weather in " + `${document.getElementById("title").innerText}` + " today is " +
```

Comment:
Could you add a comment to explain we only support the one-arg version of the method?

Reply:
I'm not entirely sure what you mean, could you explain more?